hedera-mirror-rest/api/v1/openapi.yml

```
@@ -923,6 +952,54 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Contract'
+    ContractResult:
+      type: object
+      properties:
+        amount:
+          description: The
+          example: 10
+          type: number
+        bloom:
+          description: The
+          example: "0x549358c4c2e573e02410ef7b5a5ffa5f36dd7398"
+          type: string
+        call_result:
+          description: The result returned by the function
+          example: "0x2b048531b38d2882e86044bc972e940ee0a01938"
+          type: string
+        created_contract_ids:
+          description: The network's released supply of hbars in tinybars
+          example:
+            - "0.0.1003"
+          type: array
+          items:
+            type: string
+        error_message:
+          description: The message when an error occurs during smart contract execution
+          example: "3999999999999999949"
+          type: string
+        from:
+          $ref: '#/components/schemas/SolidityAddress'
+        function_parameters:
+          description: The parameters passed to the function
+          example: "0xbb9f02dc6f0e3289f57a1f33b71c73aa8548ab8b"
+          type: string
+        gas_limit:
+          description: The maximum units of gas allowed for contract execution
+          example: 100000
+          type: integer
```

`gas_limit` and `gas_used` are modeled as bigints so we should set them similarly
```suggestion
          type: int64
```

---

Updated with
```         
          format: int64
          type: integer
```